PEAR/RunTest.php

```diff
@@ -129,7 +129,8 @@ function system_with_timeout($commandline, $env = null, $stdin = null)
         while (true) {
             /* hide errors from interrupted syscalls */
             $r = $pipes;
-            $e = $w = null;
+            unset($r[0]);
```

Comment:
The need for this would be a bit more obvious if it were placed directly after the `fclose($pipes[0])` above.